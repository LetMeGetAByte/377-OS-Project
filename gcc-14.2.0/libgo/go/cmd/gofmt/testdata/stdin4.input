version https://git-lfs.github.com/spec/v1
oid sha256:6055ef80d9d8558554e69a2937fa0e1c6b797399e14bdb2221877e16ef990197
size 39
