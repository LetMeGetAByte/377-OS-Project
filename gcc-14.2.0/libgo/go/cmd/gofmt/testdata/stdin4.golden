version https://git-lfs.github.com/spec/v1
oid sha256:5401b768fcd7e42e6adb102a3ae0a7d5415ed182bbf41b87449d105002f1fff3
size 38
